app/views/partials/_commentCard.html.erb

```diff
@@ -1,9 +1,18 @@
 <div class="horizontal-panel space-y flex flex-col border-black">
   <div class="flex flex-col px-5">
-   <% (1..items).each_with_index do |index| %>
-    <p class="text-lg">
-      Username: Comment <%= index %>
-    </p>
+    <% if comments.any? %>
+      <% comments.each do |comment| %>
+        <div class="flex items-center">
+          <p class="text-lg font-bold mr-5">
+            <%= comment.commenter || "Anoymous User" %> : 
```

Comment:
- [ ] In the previous review, this is issue was pointed out and it seems it is not addressed properly. It seems the only correction you made is to add `|| "Anonymous User"` logic to remedy the situation. This should not be the case, the design of this app doesn't allow anonymous users to add comments(`Comments should belong to a user`).
  
  ![image](https://user-images.githubusercontent.com/45002654/150382404-26ea3e22-ba52-487c-a5bd-5ee4d9eaa442.png)
  
  What we see in the screenshot above ☝🏿  should not be the case.

  The problem lies here => `comment.commenter`. There is no property called `commenter` in any `comment` object. Please refactor as follows to achieve the desired result that the first review was after.
  ```ruby
  <%= comment.author.name %> : 
  ```